code/iaas/service-discovery/api/src/main/java/io/cattle/platform/servicediscovery/api/dao/ServiceConsumeMapDao.java

```diff
@@ -12,4 +12,6 @@
     List<? extends ServiceConsumeMap> findConsumedServices(long serviceId);
 
     List<? extends ServiceConsumeMap> findMapsToRemove(long serviceId);
+    
+    List<? extends ServiceConsumeMap> findConsumedByServices(long serviceId);
```

Comment:
Is `findConsumingServices` a more appropriate name for this method?
Also, whether you change this name or not, this method and `findConsumedServices` are so similarly named, I think they warrant have a javadoc clarifying their intention. A couple one-liners like:
> Finds all the services that consume the specified service.
and
> Finds all the services that the specified service consumes.
would suffice.